doc/omnipytent.txt

```diff
@@ -25,6 +25,7 @@ Writing tasks:               |omnipytent-tasks-syntax|
                             |omnipytent-task-context-object|
 Configuration:               |g:omnipytent_defaultPythonVersion|
                             |g:omnipytent_filePrefix|
+                            |g:omnipytent_projectRootMarkers|
```

Comment:
Just a general remark: I find it a little bit annoying that the tags are not aligned in this cheat sheet. I do notice that the indentation is consistent, but it does not feel intuitive to me. Would you mind if I were to open a PR and suggest a different way of aligning? (Either perfectly aligned, or add 2 spaces for the following tags instead of subtracting 1 space).

Reply:
![2017-12-30_00-42-52](https://user-images.githubusercontent.com/1149255/34448854-7568c662-ecfa-11e7-9db6-ef60cd34df78.gif)

Reply:
Strange. This is not the case for me, see screenshot.
![screenshot](https://user-images.githubusercontent.com/998898/34448888-66c176b6-ecf2-11e7-82a1-e07fa11f298e.png)

Reply:
I'll have to bisect my plugins to see what causes this then...